vaadin-checkbox-flow-parent/vaadin-checkbox-flow/src/main/java/com/vaadin/flow/component/checkbox/Checkbox.java

```diff
@@ -305,6 +324,62 @@ public boolean isIndeterminate() {
         return getElement().getProperty("indeterminate", false);
     }
 
+    /**
+     * Sets whether the checkbox is marked as input required.
+     *
+     * @since 24.5
+     * @param required
+     *            the boolean value to set
+     */
+    public void setRequired(boolean required) {
+        getElement().setProperty("required", required);
+        this.required = required;
```

Comment:
Do we really need this alias given that `setRequiredIndicatorVisible` is present and appears to be the recommended method?